c2corg_ui/templates/index.html

```diff
@@ -1,4 +1,53 @@
+<%!
+from c2corg_common.attributes import activities
+%>
 <%inherit file="base.html"/>
 
-<h1 translate class="welcome-title text-center">Welcome to Camptocamp.org</h1>
-<app-map></app-map>
+<div class="home-page">
+  <section ng-hide="userCtrl.auth.isAuthenticated()" class="site-intro not-logged">
+    <div class="intro-banner">
+      <h1 class="welcome-title text-center" translate>The mountain sports community</h1>
+
+      <div class="route-activities text-center">
+        % for activity in activities:
+        <a href="${request.route_url('outings_index')}#&act=${activity}">
+          <span class="activity icon-${activity}" uib-tooltip="{{'${activity}' | translate}}"></span>
```

Comment:
I think ``{{'${activity}' | translate}}`` will add ``${activity}`` to Transifex because of the quotes => the extractor will think it's a standard string, not a variable.
You probably have to use ``mainCtrl.translate()`` here instead.